src/constants.js

```diff
@@ -0,0 +1,2 @@
+export const Amplitude_WS = "wss://amplitude-rpc.dwellir.com"
+export const Pendulum_WS = "wss://pendulum-rpc.dwellir.com"
```

Comment:
Let's change those to our 'official' websocket URL's hoping/assuming that they work as expected.

Reply:
Or we could also think about defining this as an array of WS URLs and then adding some fallback logic in case the client fails to connect to one of them. 
```
export const Amplitude_WS = ["wss://rpc-amplitude.pendulumchain.tech", "wss://amplitude-rpc.dwellir.com"]
export const Pendulum_WS = ["wss://rpc-pendulum.prd.pendulumchain.tech", "wss://pendulum-rpc.dwellir.com"]
```

But maybe that's too much effort for this service.

Reply:
Added the retry logic [here](https://github.com/pendulum-chain/referendum-monitoring/pull/1/commits/c440852b8a89ca49111768d41969309ee7653b6a#diff-2fb5dc3672ef759635f4f4a24db2d0cf8361ecaf020dc7448c6e085ef1555253R17)